src/App/Command/SlackNotifierCommand.php

```diff
@@ -735,15 +721,15 @@ protected function checkStatsQA(): string
         $graphQLQuery = new Query();
         $slackMessage = ':chart_with_upwards_trend: PR Stats for QA :chart_with_upwards_trend:' . PHP_EOL;
 
-        $searchPR176 = 'repo:PrestaShop/PrestaShop is:pr is:open label:1.7.6.x ' . Query::LABEL_WAITING_FOR_QA . ' -' . Query::LABEL_WAITING_FOR_AUTHOR . ' -' . Query::LABEL_WAITING_FOR_DEV . ' -' . Query::LABEL_WAITING_FOR_PM;
-        $graphQLQuery->setQuery($searchPR176);
-        $countPR176 = $this->github->countSearch($graphQLQuery);
-
         $searchPR177 = 'repo:PrestaShop/PrestaShop is:pr is:open label:1.7.7.x ' . Query::LABEL_WAITING_FOR_QA . ' -' . Query::LABEL_WAITING_FOR_AUTHOR . ' -' . Query::LABEL_WAITING_FOR_DEV . ' -' . Query::LABEL_WAITING_FOR_PM;
         $graphQLQuery->setQuery($searchPR177);
         $countPR177 = $this->github->countSearch($graphQLQuery);
 
-        $searchPRDevelop = 'repo:PrestaShop/PrestaShop is:pr is:open -label:1.7.7.x -label:1.7.6.x ' . Query::LABEL_WAITING_FOR_QA . ' -' . Query::LABEL_WAITING_FOR_AUTHOR . ' -' . Query::LABEL_WAITING_FOR_DEV . ' -' . Query::LABEL_WAITING_FOR_PM;
+        $searchPR178 = 'repo:PrestaShop/PrestaShop is:pr is:open label:1.7.8.x ' . Query::LABEL_WAITING_FOR_QA . ' -' . Query::LABEL_WAITING_FOR_AUTHOR . ' -' . Query::LABEL_WAITING_FOR_DEV . ' -' . Query::LABEL_WAITING_FOR_PM;
```

Comment:
can be configured, the develop branch have to target `develop`  it will be easier to maintain.

Reply:
Warning : the search for PR for `develop` has no label defined.

Reply:
Instead of label, use `base:` https://docs.github.com/en/github/searching-for-information-on-github/searching-issues-and-pull-requests#search-by-branch-name

Reply:
Fixed